src/Model/GalleryManager.php

```diff
@@ -13,6 +13,10 @@
 
 namespace Sonata\MediaBundle\Model;
 
+/**
+ * @phpstan-template T of GalleryInterface
+ * @phpstan-implements GalleryManagerInterface<T>
```

Comment:
GalleryManagerInterface is not a generic in 4.x but was made as generic in 
https://github.com/sonata-project/SonataMediaBundle/pull/2035/files#diff-36a18f0b194491c1f64cdf74f503a7f8c4c7c0fd72b2ff3f08d6ab77b5e65dc2R21-R25

@jordisala1991 just did 
https://github.com/sonata-project/SonataMediaBundle/blame/4.x/src/Model/GalleryManagerInterface.php#L18-L20
and then there was no need for generic for every manager related classes.